src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -494,6 +502,37 @@ class DydxChainTransactions implements AbacusDYDXChainTransactionsProtocol {
     }
   }
 
+  async signCompliancePayload(params: {
```

Comment:
[q] is it okay to declare a var that shares a namespace from an import?

Reply:
technically ok since this is a class method, but you are right probably best to avoid confusion.